tee-worker/omni-executor/binance-api/src/lib.rs

```diff
@@ -194,12 +202,55 @@ impl BinanceApi for BinanceApiClient {
 	}
 }
 
+#[async_trait]
+impl BinancePaymasterApi for BinanceApiClient {
+	async fn get_symbol_price(&self, symbol: &str) -> Result<String, Error> {
+		use crate::spot_trading_api::types::SymbolPrice;
+
+		let endpoint = "/api/v3/ticker/price";
+		let mut params = HashMap::new();
+		params.insert("symbol".to_string(), symbol.to_string());
+
+		let price_info: SymbolPrice = self.make_public_get_request(endpoint, Some(params)).await?;
+		Ok(price_info.price)
+	}
+
+	async fn get_exchange_info_for_symbol(&self, symbol: &str) -> Result<Option<u8>, Error> {
```

Comment:
Should we rename this method to something like `get_asset_precision` ? Otherwise it should probably return `ExchangeInfo`.